### DISCUSSION TITLE
Enable access to email transport

### DISCUSSION THREAD

**Discussion by WoLfulus** [directus/directus#3465] [Feature Requests]
We could start exporting `transporter` in `directus/dist/mail` so users can use it for sending emails through custom endpoints. It's just easier to have an already-configured transporter available that matches Directus settings.

**Comment by rijkvanzanten** [comment]
💯 

I wanna extract that email transport into an EmailService that is a little more robust and have it be passed on to extensions too 👍

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
